Dockerfile

```diff
@@ -3,7 +3,10 @@ FROM php:7.1-fpm-alpine
 RUN apk add --no-cache --virtual .persistent-deps \
 		git \
 		icu-libs \
-		zlib
+		zlib \
+		wget \
+		ca-certificates \
```

Comment:
@20uf  wget and ca-certificates are used because the composer is installed with wget and https.